AndroidSDKCore/src/main/java/com/leanplum/internal/RequestOld.java

```diff
@@ -361,7 +361,7 @@ private static boolean attachApiKeys(Map<String, Object> dict) {
    * @param responseBody JSONObject with response body from server.
    * @param error Exception.
    */
-  protected void parseResponseBody(JSONObject responseBody, Exception error) {
+  protected void triggerCallbackManager(JSONObject responseBody, Exception error) {
```

Comment:
are we actually triggering a callback manager here?

Reply:
I changed the name to that one because the name does not show one of the main things that this method is doing - invoking the callbacks from the requests in the response, or error callbacks for all of the callbacks in the manager.

If you have a better suggestion I could change it.